frontend/src/components/pages/CampOverview/index.tsx

```diff
@@ -0,0 +1,136 @@
+import {
+  AspectRatio,
+  Box,
+  Container,
+  Flex,
+  HStack,
+  Image,
+  Tag,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import React, { useEffect, useState } from "react";
+import placeHolderImage from "../../../assets/germany.jpeg";
+import costIcon from "../../../assets/coin.svg";
+import locationIcon from "../../../assets/location.svg";
+import ageIcon from "../../../assets/person.svg";
+import { Camp } from "../../../types/CampsTypes";
+import CampsAPIClient from "../../../APIClients/CampsAPIClient";
+
+export type CampOverviewProps = {
+  campId: string;
+};
+
+const CampOverview = (): JSX.Element => {
+  const [camp, setCamp] = useState<Camp>();
+  const campId = "62c098e7b4a7a433a7622ff4"; // hardcoded for now, TODO: update this
+
+  // TODO: update the statuses
+  enum Status {
+    PUBLISHED = "Published",
+    DRAFT = "Draft",
+  }
+
+  const status = camp?.active ? Status.PUBLISHED : Status.DRAFT;
+
+  useEffect(() => {
+    const getCampInfo = async () => {
+      const campResponse = await CampsAPIClient.getCampById(campId);
+      setCamp(campResponse);
+    };
+    getCampInfo();
+  }, []);
+
+  return (
+    <>
+      <Container maxWidth="100vw">
+        <Flex marginLeft="80px" marginRight="80px">
+          <Box width="100%" mt="1rem">
+            <HStack width="100%" marginBottom="8px" alignItems="center">
+              <Text align="left" textStyle="displayXLarge" marginBottom="8px">
+                {camp?.name}
```

Comment:
related to the previous comment about handling null campResponse/errors, we won't need the ? potentially